src/views/GroupTable/GroupTable.tsx

```diff
@@ -49,6 +50,17 @@ const GroupTable: FC<Props> = props => {
 	const currentGroup = userGroups.find(
 		(x: UserGroupState) => x.groupId.toString() === tableView.selectedGroupId.toString()
 	);
+
+	let userTable = <UsersTable users={filteredUsersTable}
+								onUserClicked={(predictionId: number) => {
+								history.push(`/prediction/${predictionId}`);
+							}} 
+					/>;
+
+	if (filteredUsersTable.length === 0) {
```

Comment:
I think that now there will be a situation that if I click on a group to enter the table - while it is fetching the group from the server, it will show "Oops, no matching users" and then after 1-2 seconds it will show the table. so it is not a very nice. you can simulate this by adding "sleep" in the API or in the mock (using setTimeout for example).

we need to add a spinner and show it while fetching the table (you can see a spinner implementation in the user prediction view) - and show the "not found" message only if we are not fetching data.